src/plot_api/plot_api.js

```diff
@@ -190,8 +190,7 @@ Plotly.plot = function(gd, data, layout, config) {
 
         return Lib.syncOrAsync([
             subroutines.layoutStyles,
-            drawAxes,
-            initInteractions
+            drawAxes
```

Comment:
Yeah, I guess initialising interactions after pushing the margins makes sense :+1: